tests/Serilog.Sinks.RabbitMQ.Tests/Approval/Serilog.Sinks.RabbitMQ.approved.txt

```diff
@@ -0,0 +1,111 @@
+        public static Serilog.LoggerConfiguration RabbitMQ(this Serilog.Configuration.LoggerSinkConfiguration loggerConfiguration, string amqpUri, string exchange = null, string exchangeType = null, Serilog.Sinks.RabbitMQ.RabbitMQDeliveryMode deliveryMode = 1, string routeKey = null, ushort heartbeat = 0, int batchPostingLimit = 0, System.TimeSpan period = default, Serilog.Formatting.ITextFormatter formatter = null, bool autoCreateExchange = false, int maxChannels = 64) { }
+        public static Serilog.LoggerConfiguration RabbitMQ(
+                    this Serilog.Configuration.LoggerAuditSinkConfiguration loggerAuditSinkConfiguration,
+                    string hostname,
+                    string username,
+                    string password,
+                    string exchange = null,
+                    string exchangeType = null,
+                    Serilog.Sinks.RabbitMQ.RabbitMQDeliveryMode deliveryMode = 1,
+                    string routeKey = null,
+                    int port = 0,
+                    string vHost = null,
+                    ushort heartbeat = 0,
+                    bool sslEnabled = false,
+                    string sslServerName = null,
+                    System.Security.Authentication.SslProtocols sslVersion = 0,
+                    System.Net.Security.SslPolicyErrors sslAcceptablePolicyErrors = 0,
+                    bool sslCheckCertificateRevocation = false,
+                    Serilog.Formatting.ITextFormatter formatter = null,
+                    bool autoCreateExchange = false,
+                    int maxChannels = 64) { }
+        public static Serilog.LoggerConfiguration RabbitMQ(
+                    this Serilog.Configuration.LoggerSinkConfiguration loggerConfiguration,
+                    string hostname,
+                    string username,
+                    string password,
+                    string exchange = null,
+                    string exchangeType = null,
+                    Serilog.Sinks.RabbitMQ.RabbitMQDeliveryMode deliveryMode = 1,
+                    string routeKey = null,
+                    int port = 0,
+                    string vHost = null,
+                    ushort heartbeat = 0,
+                    bool sslEnabled = false,
+                    string sslServerName = null,
+                    System.Security.Authentication.SslProtocols sslVersion = 0,
+                    System.Net.Security.SslPolicyErrors sslAcceptablePolicyErrors = 0,
+                    bool sslCheckCertificateRevocation = false,
+                    int batchPostingLimit = 0,
+                    System.TimeSpan period = default,
+                    Serilog.Formatting.ITextFormatter formatter = null,
+                    bool autoCreateExchange = false,
+                    int maxChannels = 64) { }
+    }
+}
+namespace Serilog.Sinks.RabbitMQ
```

Comment:
@ArieGato As you can see there are a lot of public APIs here that should not be exposed.